src/components/ProviderContainer.tsx

```diff
@@ -33,6 +35,9 @@ const SBComponent = ({ children }: { children: React.ReactElement }) => {
     serviceName,
     isMobileView,
   } = useConstantState();
+  const { botStyle } = useWidgetSetting();
+  const session = useWidgetSession();
```

Comment:
minor: can we add
```
const { userId,  sessionToken } = session;
```

Reply:
This is my personal preferred style.
For short variables, I do not like destructuring or expanding them unnecessarily